### ISSUE TITLE
mktap web -p doesn't accept windows absolute paths

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11408]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| moonfallen reported|
|-|-|
|Trac ID|trac#478|
|Type|defect|
|Created|2004-01-16 00:34:15Z|

<details><summary>Searchable metadata</summary>

```
trac-id__478 478
type__defect defect
reporter__moonfallen moonfallen
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1074213255000000 1074213255000000
changetime__1470702477811252 1470702477811252
version__ 
owner__ 
cc__moonfallen
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Has something to do with parsing for port numbers (why is it doing that in the
-p argument?)  Really, this says it all:

C:\Documents and Settings\cory\Desktop>mktap web -p "C:\documents and settings\c
ory\desktop"
Traceback (most recent call last):
  File "C:\Python22\scripts\mktap.py", line 31, in ?
    run()
  File "C:\Python22\Lib\site-packages\twisted\scripts\mktap.py", line 160, in ru
n
    options.subOptions)
  File "C:\Python22\Lib\site-packages\twisted\scripts\mktap.py", line 58, in mak
eService
    ser = mod.makeService(options)
  File "C:\Python22\Lib\site-packages\twisted\tap\web.py", line 188, in makeServ
ice
    strports.service(config['port'], site).setServiceParent(s)
  File "C:\Python22\Lib\site-packages\twisted\application\strports.py", line 168
, in service
    name, args, kw = parse(description, factory, default)
  File "C:\Python22\Lib\site-packages\twisted\application\strports.py", line 152
, in parse
    return (args[0].upper(),)+_funcs[args[0]](factory, *args[1:], **kw)
KeyError: C
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
nm, should have been using "--path" not -p.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset cfd85e530c09e35be6e211176b80772eb8e7ef85
```
#!CommitTicketReference repository="" revision="cfd85e530c09e35be6e211176b80772eb8e7ef85"
Merge pull request #478 from twisted/8734-rodrigc-tcp-py3

Author: hawkowl, rodrigc
Reviewer: alex
Fixes: #8734

Fix use of errno in twisted.internet.test.test_tcp on Python 3
```
